rfd/0215-bot-details-web.md

```diff
@@ -0,0 +1,218 @@
+## Why
+
+Management operations and diagnostic information on bots is only possible via `tclt` - this change seeks to make these more accessible and more friendly for non-technical users (i.e. users less comfortable on the command line). The new page is targeted mainly at members of the Infrastructure Security team whose role it is to configure and maintain a Teleport deployment, as well as enrol protected resources.
+
+## Details
+
+### Day 1 vs. day 2
+In it's first increment the Bot Details page has no expected differences between day 1 and day 2 experiences, and it's likely to be used by users who are already acquainted with Teleport.
+
+### User stories
+
+As a member of the **Infrastructure Security team**,
+I would like to **view information about a bot** (such as name and create/updated at & by),
+So that I can track changes to the bot’s configuration for auditing or troubleshooting purposes.
+
+As a member of the **Infrastructure Security team**,
+I would like to link out to documentation about **what a bot is and how it works**,
+So that I can get the most out of the capability and feel confident deploying a solution leveraging bots.
+
+As a member of the **Infrastructure Security team**,
+I would like to **edit the assigned roles, allowed traits and max session dutration** for a bot,
+So that I can easily extend or reduce the scope of a bot without the need to migrate existing agents (e.g. by needing to recreate the bot) or using `tctl`.
+
+As a member of the **Infrastructure Security team**,
+I would like to **delete a bot** when it is no longer required,
+So that I can reduce unnecessary access paths to resources, and keep the cluster configuration tidy and free of historic clutter.
+
+As a member of the **Infrastructure Security team**,
+I would like to **lock a bot** and all instances,
+So that current and future access to protected resources is immediately prevented.
+
+As a member of the **Infrastructure Security team**,
+I would like to **view assigned roles and allowed traits** for a bot,
+So that I can easily determine the scope of access at a glance.
+
+As a member of the **Infrastructure Security team**,
+I would like to **view configured join methods** for a bot,
+So that I can easily determine the enrolment mechanisms available at a glance.
+
+As a member of the **Infrastructure Security team**,
+I would like to see a list of **currently active** instances for a bot with **last active times** and high-level characteristics (such as **bot name**, **hostname** and **join method**),
+So that I can assess the usage of the bot by scanning over the results to build an overall picture of recency and a distribution of characteristics.
+
+### UX
+
+![[0215-iteration-1.png]]
```

Comment:
I think you may need to modify these image links to be "GitHub" style markdown rather than Obsidian style